.changeset/modern-ways-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/node': patch
+---
+
+Cancel response stream when connection closes
```

Comment:
```suggestion
Fixes a bug where the response stream would not cancel when the connection closed
```
(Not blocking) I can meet you halfway! What about something of this style? (Make it accurate.)

(I did like @lilnasy 's version, but can appreciate that everyone wants something shorter for a patch!)